packages/dd-trace/src/telemetry/index.js

```diff
@@ -52,8 +52,6 @@ function flatten (input, result = [], prefix = [], traversedObjects = null) {
 
 function appStarted () {
   return {
-    integrations: getIntegrations(),
```

Comment:
TODO: Accidental commit

Reply:
need to add back integrations + dependencies